backend/src/achievements-log/dto/response-achievements-log.dto.ts

```diff
@@ -0,0 +1,7 @@
+import { OmitType } from '@nestjs/swagger';
+import { AchievementsLogDto } from './achievements-log.dto';
+
+export class ResponseAchievementsLogDto extends OmitType(
+  AchievementsLogDto,
```

Comment:
I feel like we might want to omit the log Id, as I don't think it serves a purpose for the user.